graylog2-web-interface/packages/eslint-config-graylog/index.js

```diff
@@ -50,6 +50,15 @@ module.exports = {
     'react/prefer-stateless-function': 'warn',
     'react/static-property-placement': 'off',
 
+    'padding-line-between-statements': [
+      'error',
+      { blankLine: 'always', prev: '*', next: 'return' },
+      { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
+      { blankLine: 'any', prev: ['const', 'let', 'var'], next: ['const', 'let', 'var'] },
+      { blankLine: 'always', prev: 'directive', next: '*' },
```

Comment:
Afaics, `directive` means prologue imperatives like `use strict`, which we do not use at all. Maybe we can insert blank lines after `function` definitions instead?